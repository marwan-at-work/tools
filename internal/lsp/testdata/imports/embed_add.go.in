package imports //@import("package")

import (
	"fmt"
)

//go:embed embed_add.go.in
var file embed.FS

func _() {
	fmt.Println(file)
}
