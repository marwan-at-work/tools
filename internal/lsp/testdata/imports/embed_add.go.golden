-- goimports --
package imports //@import("package")

import (
	"embed"
	"fmt"
)

//go:embed embed_add.go.in
var file embed.FS

func _() {
	fmt.Println(file)
}

